.travis.yml

```diff
@@ -1,7 +1,9 @@
 language: python
 python:
+  - 3.4                                            
+  - 3.5  
   - 3.6
-
+    
```

Comment:
Remove this unwanted whitespaces. Try keep your amount of changes as small as you can.